lib/routes.js

```diff
@@ -1,3 +1,5 @@
+var currentYear = (new Date()).getFullYear();
```

Comment:
This adds the current year to the copyright in the footer. I couldn't find any docs on HapiJS for global route variables.